config/logstash.yml

```diff
@@ -113,3 +113,12 @@
 #
 # Where to find custom plugins
 # path.plugins: []
+#
+# ------------ Slow log thresholds ---------
+#
+# An slow log threshold is a time limit for an operation that when is overcome it will
+# be reported at the related log file. Time is seconds.
+#
+# The last building block of the component is the log level (default: warn)
+#
+# filters.dns.getaddress.warn: 5
```

Comment:
I think this should be `filters.dns.threshold.warn: 5`. As a user how will I know there is `getaddress` method in the code? I want to log all the slow operations in DNS filter. the user shoudn't be required to know the sub-methods.